tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -643,7 +643,7 @@ Status MarkForCompilationPassImpl::CreateClusters() {
   // Names for each cluster.
   std::unordered_map<int, string> cluster_names;
 
-  if (debug_options_.dump_graphs) {
+  if (debug_options_.dump_graphs || VLOG_IS_ON(1)) {
```

Comment:
What do you think about making this `VLOG_IS_ON(3)`?

Reply:
I am agree with your idea to reduce the debug level  of dump file , but i am not sure which specific level to reduce to . Is there any instrument docs for this?